src/mpl/configure.ac

```diff
@@ -1080,13 +1080,15 @@ if test "$have_gpu" = "no" ; then
     PAC_PUSH_FLAG(CPPFLAGS)
     PAC_APPEND_FLAG([-D__HIP_PLATFORM_AMD__], [CPPFLAGS])
     PAC_CHECK_HEADER_LIB_OPTIONAL([hip],[hip/hip_runtime_api.h],[amdhip64],[hipStreamSynchronize])
-    PAC_POP_FLAG(CPPFLAGS)
 fi
 
 if test "X${pac_have_hip}" = "Xyes" ; then
     have_gpu="yes"
     AC_DEFINE([HAVE_HIP],[1],[Define if HIP is available])
-    AC_DEFINE([__HIP_PLATFORM_AMD__],[1],[AMD GPU HIP available])
```

Comment:
Was this broken?

Reply:
GPU_SUPPORT was missing. `__HIP_PLATFORM_AMD__` is defined by user and should be not AC_DEFINED in MPICH.